protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufResponseConverterFunctionProvider.java

```diff
@@ -40,7 +40,7 @@ public ResponseConverterFunction createResponseConverterFunction(
             Type returnType,
             ResponseConverterFunction responseConverter) {
         if (isSupportedType(returnType)) {
-            return new ProtobufResponseConverterFunction();
+            return responseConverter.orElse(new ProtobufResponseConverterFunction());
```

Comment:
Could you also do the same thing for `ScalaPbResponseConverterFunctionProvider`?
If you are unfamiliar with Scala, just let us know. We can add a commit for that to this PR. 😉

Reply:
A little optimization we can use is creating a singleton and reusing it. 😄

Reply:
> Could you also do the same thing for ScalaPbResponseConverterFunctionProvider?
If you are unfamiliar with Scala, just let us know. 

We're unfamiliar with Scala :)